app/models/miq_request_workflow.rb

```diff
@@ -54,6 +54,10 @@ def instance_var_init(values, requester, options)
     @values       = values
     @filters      = {}
     @requester    = User.lookup_by_identity(requester)
+    if (group_description = values[:requester_group])
```

Comment:
They are very similar.
in `instance_var_init`, `values` is `@values`.
But in `create_request` -> `set_request_vars`, it is a local `values` (not `@values`)